spec/myapp_spec.rb

```diff
@@ -24,9 +24,24 @@ def app
     end
 
     it 'returns shops in json' do
-
       response_data = JSON.parse(last_response.body)
       expect(response_data).to eq([{"description"=>"bla", "id"=> shop.id, "name"=>"mandy"}])
     end
   end
+
+  describe "GET v1/shops/:id" do
+    let!(:shop) { Shop.new({name: "RedPanda", description: "It is fluffy"}).create }
+
+    let(:response) { get "/v1/shops/#{shop.id}" }
+
+    it 'returns status 200 OK' do
+      expect(Shop.last).not_to eq(nil)
```

Comment:
I think you can get rid of this line

Reply:
What should I replace this expect line with?

Reply:
nothing :D